ui/lib/apps/ContinuousProfiling/pages/List.tsx

```diff
@@ -65,42 +98,41 @@ export default function Page() {
         },
       },
       {
-        name: t('continuous_profiling.list.table.columns.status'),
+        name: t('conprof.list.table.columns.status'),
         key: 'status',
         minWidth: 100,
         maxWidth: 150,
         onRender: (rec) => {
+          // The following state will be refactored later by ngm
           if (rec.state === 'failed') {
             // all failed
             return (
               <Badge
                 status="error"
-                text={t('continuous_profiling.list.table.status.failed')}
+                text={t('conprof.list.table.status.failed')}
               />
             )
           } else if (rec.state === 'success') {
             // all success
             return (
               <Badge
                 status="success"
-                text={t('continuous_profiling.list.table.status.finished')}
+                text={t('conprof.list.table.status.finished')}
```

Comment:
Update the state after PR https://github.com/pingcap/ng-monitoring/pull/50 is merged.